playwright.config.ts

```diff
@@ -26,7 +26,7 @@ const config: PlaywrightTestConfig = {
    */
   retries: process.env.CI ? 2 : 1,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : undefined,
```

Comment:
It would be worth updating this comment to indicate that our tests depend on being run one at a time due to database patterns.

Reply:
Right, I missed that. Will update that with some info on this